src/main/java/seedu/user/UserList.java

```diff
@@ -47,14 +47,23 @@ public User getUserByName(String userName) {
     }
     
     public void removeUser(String userName)  throws WhereGotTimeException {
-        System.out.println(userName);
+        boolean isMatch = false;
+        
+        if (users.isEmpty()) {
+            throw new WhereGotTimeException("No users exist!");
+        }
+        
         for (int i = 0; i < users.size(); i++) {
             System.out.println(users.get(i).getName());
             if (users.get(i).getName().equals(userName)) {
                 users.remove(i);
-            } else {
-                throw new WhereGotTimeException(userName + " does not exist!");
+                System.out.println(userName + " has been removed.");
+                isMatch = true;
             }
         }
+
+        if (isMatch == false) {
+            throw new WhereGotTimeException(userName + " does not exist!");
+        }
```

Comment:
Nice work, but is it possible to prompt user to enter password when removing an account? Otherwise it might result in poor security design.